webapp/package.json

```diff
@@ -29,7 +29,7 @@
     "react-dom": "15.4.2",
     "react-intl": "2.2.3",
     "react-router": "2.8.1",
-    "react-select": "1.0.0-rc.3",
+    "react-select": "1.0.0-rc.2",
```

Comment:
We might also want to double check that 1.0.0-rc.2 exists still. Looking at the GitHub page for that, it only shows an rc1 and an rc3